File: App/Models/DbContext/CaseSheet.cs
using App.Models.EntityModels;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Security.AccessControl;
using System.Threading.Tasks;

namespace App.Models.DbContext
{
    public class CaseSheet
    {

        [Key]
        public long Id { get; set; }
        public long PatientID { get; set; }
        [ForeignKey(nameof(PatientID))]
        public Patient Patient { get; set; }
        [Display(Name = "PRESENTING COMPLAINTS")]
        public string PRESENTING_COMPLAINTS { get; set; }
        [Display(Name = "History Of Presenting Illness")]
        public string HistoryOfPresentingIllness { get; set; }
        [Display(Name = "Past History")]
        public string PastHistory { get; set; }
        [Display(Name = "Personal History")]
        public string PersonalHistory { get; set; }
        public string Diet { get; set; }
        public string Appetite { get; set; }
        public string Sleep { get; set; }
        public string Bowel { get; set; }
        public string Bladder { get; set; }
        public string Addiction { get; set; }
        [Display(Name = "Family History")]
        public string FamilyHistory { get; set; }

        //Vitals
      
        public string BP { get; set; }
        public string PR { get; set; }
        public string RR { get; set; }
        public string Temp { get; set; }
        public string SpO2 { get; set; }

        //GENERAL EXAMINATION
       
        public string Pallor { get; set; }
        public string Icterus { get; set; }
        public string Cyanosis { get; set; }
        public string Clubbing { get; set; }
        public string Edema { get; set; }
        public string Lymphadenopathy { get; set; }

        //SYSTEMIC EXAMINATION
        [Display(Name = "Respiratory System")]
        public string RespiratorySystem { get; set; }
        public string CNS { get; set; }
        public string CVS { get; set; }
        public string PerAbdomen { get; set; }
        [Display(Name = "Locoregional Exam")]
        public string LocoregionalExam { get; set; }
        
       
        public string Remark { get; set; }
        public string Value1 { get; set; }
        public string Value2 { get; set; }
        public string Value3 { get; set; }

    }
}

File: App/wwwroot/customJs/CaseSheet.js
function getCaseSheetFromModel() {
    var models = [];

    var model = {};

    model.PatientID = $("#CaseSheet_PatientID").val(); // this is for edit
    if (model.PatientID == "") {
        model.PatientID = $("#PatientModel_PatientID").val(); //this id for create after patient is created
    }else if (model.PatientID == "") {
        model.PatientID = 0;
    }
    model.Id = $("#CaseSheet_Id").val();
    if (model.Id == "") {
        model.Id = 0;
    }
    model.PRESENTING_COMPLAINTS = $("#CaseSheet_PRESENTING_COMPLAINTS").val();
    model.HistoryOfPresentingIllness = $("#CaseSheet_HistoryOfPresentingIllness").val();
    model.PastHistory = $("#CaseSheet_PastHistory").val();
    model.PersonalHistory = $("#CaseSheet_PersonalHistory").val();
    model.Diet = $("#CaseSheet_Diet").val();
    model.Appetite = $("#CaseSheet_Appetite").val();
    model.Sleep = $("#CaseSheet_Sleep").val();
    model.Bowel = $("#CaseSheet_Bowel").val();
    model.Bladder = $("#CaseSheet_Bladder").val();
    model.Addiction = $("#CaseSheet_Addiction").val();
    model.FamilyHistory = $("#CaseSheet_FamilyHistory").val();
   //Vitals
    model.BP = $("#CaseSheet_BP").val();
    model.PR = $("#CaseSheet_PR").val();
    model.RR = $("#CaseSheet_RR").val();
    model.Temp = $("#CaseSheet_Temp").val();
    model.SpO2 = $("#CaseSheet_SpO2").val();

    //GENERAL EXAMINATION
    model.Pallor = $("#CaseSheet_Pallor").val();
    model.Icterus = $("#CaseSheet_Icterus").val();
    model.Cyanosis = $("#CaseSheet_Cyanosis").val();
    model.Clubbing = $("#CaseSheet_Clubbing").val();
    model.Edema = $("#CaseSheet_Edema").val();
    model.Lymphadenopathy = $("#CaseSheet_Lymphadenopathy").val();

    //SYSTEMIC EXAMINATION
    model.RespiratorySystem = $("#CaseSheet_RespiratorySystem").val();
    model.CNS = $("#CaseSheet_CNS").val();
    model.CVS = $("#CaseSheet_CVS").val();
    model.PerAbdomen = $("#CaseSheet_PerAbdomen").val();
    model.LocoregionalExam = $("#CaseSheet_LocoregionalExam").val();
    
    models.push(model);
     $("#CaseSheetModelsJson").val(JSON.stringify(models));
    return true;
}

$("#CaseSheetBtn").click(function (event) {
    getCaseSheetFromModel();
    event.preventDefault();
    // var formData = $(this).serialize();
    var modelsJson = $("#CaseSheetModelsJson").val();
    // Submit the form using AJAX
    $.ajax({
        type: "POST",
        url: "/Patient/CaseSheet",
        data: {
            model: modelsJson
        },
        contentType: "application/x-www-form-urlencoded",
        dataType: "json",

        success: function (msg) {
            $("#CaseSheetMessage").text(msg).show();
            $("#CaseSheetBtn").val("Successfull").prop("disabled", true);
        },
        error: function (xhr, status, error) {
            // Handle the error response
        }
    });

    $('#AddCaseSheetForm input[type=text]').each(function () {
        $(this).val('');
    });
    $('#AddCaseSheetForm textarea').each(function () {
        $(this).val('');
    });
});